src/api/__tests__/Groups.test.js

```diff
@@ -0,0 +1,60 @@
+import Groups from '../Groups';
+
+let groups;
+const BASE_URL = 'https://www.foo.com';
+const FILE_ID = 'foo';
+const id = 123;
+describe('api/Groups', () => {
+    beforeEach(() => {
+        groups = new Groups({});
+    });
+
+    describe('CRUD operations', () => {
+        const file = {
+            id: 'foo',
+            permissions: {},
+        };
+        const groupId = '123';
+        const group = {
+            id: groupId,
+        };
+
+        const successCallback = jest.fn();
+        const errorCallback = jest.fn();
+
+        beforeEach(() => {
+            // mock out groups.get response
+            groups.get = jest.fn();
+            groups.checkApiCallValidity = jest.fn(() => true);
+            groups.getBaseApiUrl = jest.fn(() => BASE_URL);
+        });
+
+        describe('getGroupCount()', () => {
+            test('should get a well formed', () => {
+                const expectedRequestData = {
+                    data: {
+                        limit: 1,
+                    },
+                };
+
+                groups.getGroupCount({
+                    file,
+                    group,
+                    successCallback,
+                    errorCallback,
+                });
+
+                // getGroupCount() returns Promise
+                // Verify whether groups.get resolves or rejects
+                // cannot use toBeCalledWith()
+                expect(groups.get).toBeCalledWith({
```

Comment:
This test currently fails because of the specified comments. Need to update test that handles when groups.get resolves/rejects. Refer to `getGroupCount` implementation in `Groups.js` for context

Reply:
Let's use [`jest.any`](https://jestjs.io/docs/en/expect#expectanyconstructor) here, on the `successCallback` and `errorCallback` lines. You'll want to check to see if some function is called.